### ISSUE TITLE
App repeatedly resets to "All logins" view soon after startup/unlock

### ISSUE THREAD

**Issue by SimonSapin** [mozilla-lockwise/lockwise-android#892]
## Steps to reproduce

* (If the app is already open, close it)
* Open the app and unlock it. The "All logins" view shows up
* Tap on one of the shown logins, or on the search icon

### Expected behavior

The details of that login show up, or the search bar and keyboard show up, respectively

### Actual behavior

The expected behavior does happen, but after then after less than one second the app goes back to the "All logins" view. If trying again, the same may happen again for many seconds after startup. Only after a while does the app seem to "stabilize". It is effectively unusable in the meantime.

### Device & build information

* Device: Pixel 2
* Build version: 1.1.3 from Play Store

### Notes

Possibly related to (or a duplicate of) https://github.com/mozilla-lockwise/lockwise-android/issues/763, except that I’m not explicitly triggering a sync. Or to https://github.com/mozilla-lockwise/lockwise-android/issues/762, except that the problem stops some time after startup.

**Comment by 0xdec** [comment]
I can reproduce this on my OnePlus 5T running OxygenOS 9.0.7. It renders the app completely unusable, except to autofill in form fields.

**Comment by SimonSapin** [comment]
> completely unusable

Not for me, fwiw. I "only" have to wait 5 to 20 seconds or so after startup.

**Comment by krzyk** [comment]
For me it is unusable also, I wasn't able to stabilize it (Pixel 3, Android 10).

**Comment by tncga** [comment]
I can reproduce same problem on my Asus Zenfone 5z running Android 9 and Lockwise v1.1.3 from Google Play Store. Also, autofill does not show up sometimes.

**Comment by jhugman** [comment]
Duplicate of #987. Closing.
